### ISSUE TITLE
Diagram

### ISSUE THREAD

**Issue by antoniogarzaq** [oelizondo/learn-git#2]
A simple diagram that shows the hierarchy/order of the master dev and branches could be very helpful for a first timer. These concepts are similar but must be understood in order to have a good idea of how github works. Therefore clarifying it with a visual aid could go a long way in clearing up any doubts.

**Comment by jesusalatorre** [comment]
Agreed, a simple drawing to represent how these branches interact can be very helpful for understanding commands like merge and push more easily, as well as the overall importance of Dev and master.
